kubernetes/linux/setup.sh

```diff
@@ -33,7 +33,7 @@ sed -i -e 's/# en_US.UTF-8 UTF-8/en_US.UTF-8 UTF-8/' /etc/locale.gen && \
 
 #install oneagent - Official bits (3/14/2022)
 if [ "${ARCH}" != "arm64" ]; then
-    wget "https://github.com/microsoft/Docker-Provider/releases/download/1.17.0/azure-mdsd_1.17.0-build.master.354_x86_64.deb" -O azure-mdsd.deb
+    wget "https://github.com/microsoft/Docker-Provider/releases/download/testing%2Fmdsd%2F10062022/azure-mdsd_1.23.0-build.master.12_x86_64.deb" -O azure-mdsd.deb
 else
     wget "https://github.com/microsoft/Docker-Provider/releases/download/1.17.1-arm64-master/azure-mdsd_1.17.1-build.master.366_aarch64.deb" -O azure-mdsd.deb
```

Comment:
so we will be using the same build for AMD64 and Arch64 versions, right?

Reply:
yes, this is temporary link. will update with the new builds for both arm64 and amd64